scripts/download_LSMDC.sh

```diff
@@ -0,0 +1,47 @@
+# Download 44 movies from LSMDC dataset
```

Comment:
@LoreggiaGiacomo  this file should be removed

Reply:
Done